src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/enums/SessionConflict.kt

```diff
@@ -3,5 +3,5 @@ package uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.vi
 @Suppress("unused")
 enum class SessionConflict {
   NON_ASSOCIATION,
-  DOUBLE_BOOKED,
+  DOUBLE_BOOKING_OR_RESERVATION,
```

Comment:
has this been updated everywhere? Seems like the change is only here?

Reply:
Yes at the look of it,.. I assume the tests are not testing for it,.. you would only be testing the wiring anyhow, as there is no logic to test.